client/src/pages/competitions/[id].tsx

```diff
@@ -1,31 +1,170 @@
+import {
+  Grid,
+  Container,
+  Avatar,
+  Stack,
+  Paper,
+  Typography,
+} from '@mui/material'
+import { AxiosError } from 'axios'
 import { GetServerSideProps } from 'next'
+import api from '@/api'
+import { requestOption } from '@/api/requestOption'
 import CompetitionPageBase from '@/components/competitionPageBase'
+import { MarkDown } from '@/components/markdown'
+
+type DateProps = {
+  startAt: string
+  endAt: string
+}
+
+type AuthorProps = {
+  name: string
+  iconUrl: string
+}
+
+type RuleProps = {
+  rule: string
+}
 
-// 仮の構造
 type Props = {
-  competitionId: string
+  id: string
+  name: string
+  dateProps: DateProps
+  authorProps: AuthorProps
+  ruleProps: RuleProps
+}
+
+// 開催期間
+const DatePaper = (props: DateProps) => {
+  let startStr = new Date(props.startAt).toLocaleString()
+  let endStr = new Date(props.startAt).toLocaleString()
+  return (
+    <Paper
+      sx={{
+        p: 2,
+        display: 'flex',
+        flexDirection: 'column',
+        height: 120,
+      }}
+    >
+      <Typography variant='h6' color='primary' gutterBottom>
+        開催期間
+      </Typography>
+      <Stack direction='row' spacing={2} alignItems='center'>
+        <Typography>
+          {startStr} &gt; {endStr}
```

Comment:
開催期間、だからこの表示の方がわかりやすいかな？
```suggestion
          {startStr} ~ {endStr}
```